README.md

```diff
@@ -1 +1,34 @@
-# TeamC-API
\ No newline at end of file
+# TeamC-API
+
+Database
+---
+1. The api connects to database: AgileSprints_SamB
+
+Config
+---
+1. Create an empty file called employeedb.properties in <directory_clone_was_ran>/TeamC-API/
```

Comment:
Can just call this the project root instead of the path

Reply:
done